### PR TITLE
Feature/137 listings

### PR DESCRIPTION
**Pull request by chris-lawton**
front end work for the news, lab, fund, project and person listing pages

### CODE REVIEW

**Comment by todd-dembrey** [comment]
The lsiting pages look pretty similar to me. I would prefer to see the listing pages sharing a base template with a couple of blocks for where they differ.

**Comment by chris-lawton** [comment]
Created a listing index page template, which fund, lab and project index pages now all extend from - https://github.com/OpenTechFund/opentech.fund/pull/82/commits/1fd199a78eda54df5824fab77475e504d8d7bab0

**Comment by chris-lawton** [comment]
@todd-dembrey - can you review https://github.com/OpenTechFund/opentech.fund/pull/82/commits/22925bf84f39af81e8820fb899fb895987707105 please

File: opentech/public/people/templates/people/includes/person_listing.html [hunk 1]
```diff
@@ -1,7 +1,13 @@
-<article class="person">
-    <h2><a href="{% pageurl person %}">{{ person.first_name }} {{ person.last_name }}</a></h2>
-
+{% load wagtailcore_tags wagtailimages_tags %}
```
Comment (hippogriffic) [inline opentech/public/people/templates/people/includes/person_listing.html]: I would usually leave a blank line between load or extend tags and the template content, just for readability. Not mandatory though

**Pull request merged**